code/game/machinery/rechargestation.dm

```diff
@@ -92,6 +92,12 @@
 		A.blob_act()
 		qdel(src)
 
+/obj/machinery/recharge_station/narsie_act()
+	go_out()
+	new /obj/effect/gibspawner/generic(get_turf(loc)) //I REPLACE YOUR TECHNOLOGY WITH FLESH!
+	qdel(src)

```

Comment:
Similar things where the issue could occur: Cryo tubes, sleepers, advanced body scanners, genetics scanners, morgue trays, closets, disposal bins, exosuits - not sure what the ruling on the last one should be